README.md

```diff
@@ -139,6 +139,12 @@ You can also specify separate download roots for npm and node as they are now st
 </plugin>
 ```
 
+By default node/npm will be cached to to the Maven local repository. To disable this and use the `workingDirectory/cache` add
```

Comment:
What do you mean workingDirectory/cache? Will a cache directory appear there by default? (I haven't used the cache functionality myself)